cloudslang-tests/src/test/resources/yaml/get_function_test_flow_basic.sl

```diff
@@ -14,26 +14,26 @@ flow:
   name: get_function_test_flow_basic
   inputs:
     - input1:
-        default: None
+        default: null
```

Comment:
I think you should put a non existing param in order to keep the functionality. e.g:
```yaml
- input1:
    default: ${None}
```